widget-reference/filepicker.md

```diff
@@ -36,7 +36,7 @@ You can upload files by creating a post API and referring to the base64 version
       </td>
     </tr>
     <tr>
-      <td style="text-align:left"><b>uploadedFileUrls</b>
+      <td style="text-align:left"><b>uploadedFileURLs</b>
```

Comment:
This is the name of the property in the codebase, we can't update it without changing the property